e2e/paths/enterAllocationProposal_test.js

```diff
@@ -1,3 +1,5 @@
+/* global xScenario */
```

Comment:
That should not be needed. Make sure you are aligned with master and you have done yarn install to get latest linter.

Reply:
👍